docs/source/using/experimental-features.adoc

```diff
@@ -151,3 +151,51 @@ To get the status of the DNS server:
 ----
 $ minishift dns status
 ----
+
+[[run-against-an-existing-machine]]
+== Run against an Existing Machine
```

Comment:
This note applies generally.

I'd like for the machines referenced here to be explicit. For instance, at the moment, we refer to "an existing machine" -- this should be "an existing virtual machine" to help differentiate it from the host machine.